.github/workflows/docs.yml

```
@@ -55,7 +55,7 @@ jobs:
 
   deploy:
     name: Deploy docs to GitHub Pages
-    if: github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/3.0_develop')
+    if: github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/3.0_develop' || github.ref == 'refs/heads/doc_week')
```

```suggestion
    if: github.event_name == 'push'
```

Hm, @ketan96-m sorry, I guess what I suggested was both insufficient and redundant as

https://github.com/ssl-hep/ServiceX_frontend/blob/2dfb069200bf80572941fe78893684f3b859e7af/.github/workflows/docs.yml#L3-L7

already covers the run triggers for `push` events and so specifying them here again is a bit redundant unless you want to guard against people triggering things by adding in branches to the `push` block. But for you all that's probably fine, so you could just remove the `github.ref` logic here all together and then add `doc_week` to the branch list in

https://github.com/ssl-hep/ServiceX_frontend/blob/2dfb069200bf80572941fe78893684f3b859e7af/.github/workflows/docs.yml#L3-L7